src/screenControl.py

```diff
@@ -100,6 +105,38 @@ def getIsSidebarMode(self):
         (maxy, maxx) = self.screenControl.getScreenDimensions()
         return maxx > 200
 
+    def outputDescriptionPane(self, lineObj):
+        if not self.getIsSidebarMode():
+            return
+        (maxy, maxx) = self.screenControl.getScreenDimensions()
+        borderX = maxx - self.WIDTH
+        startY = self.SIDEBAR_Y + 1
+        headerLine = 'Description for ' + lineObj.path + ' :'
+        descLines = [
+                        lineObj.getTimeLastAccessed(),
+                        lineObj.getTimeLastModified(),
+                        lineObj.getOwnerId(),
+                        lineObj.getSizeInBytes(),
+                        lineObj.getLengthInLines()
+                    ]
+        self.printer.addstr(startY, borderX + 2, headerLine)
+        y = startY + 2
+        for descLine in descLines:
+            self.printer.addstr(y, borderX + 2, '    * ' + descLine)
```

Comment:
i guess the size of the line is somewhat fixed, but theoretically if we had a SUPER long line it could print off the screen. and i think that results in a fatal but i dont entirely remember.

do you mind testing and seeing what happens? we might want to add a trim here just in case